### DISCUSSION TITLE
Actividad 1 - GRUPO 14

### DISCUSSION THREAD

**Discussion by JuanbyGuada** [frlp-utn-ingsoft/catedra#87] [Show and tell]
https://github.com/JuanbyGuada/vetsoft

Casas Ortiz de Rosas, Juan Bautista.
Tastaca Lugo, Aylen.

**Comment by RodrigoJacznik** [comment]
Buenas!
La actividad está muy bien resuelta, queda aprobada. Siguieron bien feature branch.
Muy bueno y completo el trabajo 👏 👏

Tengan cuidado con mergear con conflictos. Por ejemplo el commit d4d61

Saludos
